src/components/dev-hub/academia-sign-up-form.js

```diff
@@ -0,0 +1,200 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { colorMap, size, screenSize } from './theme';
+import { P, ArticleH3, H3 } from './text';
+import Input from './input';
+import Button from './button';
+import Select from './select';
+import Checkbox from '@leafygreen-ui/checkbox';
+import SectionHeader from './section-header';
+import SuccessState from './success-state';
+
+const StyledButton = styled(Button)`
+    display: flex;
+    margin: ${size.large} auto;
+`;
+
+const StyledSelect = styled(Select)`
+    color: ${colorMap.greyLightTwo};
+`;
+
+const ErrorMessage = styled(P)`
+    color: ${colorMap.salmon};
+`;
+
+const StyledSectionText = styled(ArticleH3)`
+    color: ${colorMap.greyLightTwo};
+    font-family: akzidenz;
+    font-weight: normal;
+`;
+
+const InstructorSection = styled('div')`
+    display: grid;
+    grid-template-columns: repeat(auto-fill, 335px);
+    grid-row-gap: ${size.mediumLarge};
+    justify-content: space-between;
+    margin-bottom: ${size.xlarge};
+    @media ${screenSize.upToMedium} {
+        display: block;
```

Comment:
the css grid is NICE! Looks 10x better already

On mobile we will need to update these styles a bit to let the grid work properly. The shape is now one column and three rows, can we update this media query to use this new layout instead of `block`? That way we will also get some nice spacing between form elements